lib/server.js

```diff
@@ -28,21 +45,76 @@ app.get('/people/:id', (req, res, next) => {
   res.json(record[0]);
 });
 
+/**
+ * @route POST /api/v1/:model
+ * Model must be a proper model, located within the ../models folder
+ * @param {model} model.path.required
+ * @returns {object} 200 - Count of results with an array of results
+ * @returns {Error}  500 - Unexpected error
+ */
+
 // Route to Create a person
 app.post('/people', (req, res, next) => {
   let record = req.body;
-  // what can you do to ensure there are no
-  // duplicate ids?
-  record.id = Math.random();
+  record.id = db.teams.length+=100;
   db.people.push(record);
   res.json(record);
 });
 
 // Route to Update a person
-app.put('/people/:id', (req, res, next) => {});
+app.put('/people/:id', (req, res, next) => {
+  let id = parseInt(req.params.id);
+  let record = db.people.filter(record => record.id === id);
+  record[0].firstName = req.body.firstName;
+  record[0].lastName = req.body.lastName;
+  record[0].birthday = req.body.birthday;
+  record[0].team = req.body.team;   
+  record[0].likes = req.body.likes;
+  
+  res.json(record);
```

Comment:
You're returning the updated record, but not actually saving it back into the `db`